tests/auto-jenkins/start_rt_auto_jenkins.sh

```diff
@@ -22,9 +22,10 @@ elif [[ $HOSTNAME == tfe* ]]; then
 elif [[ $HOSTNAME == gaea* ]]; then
   export PATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/envs/ufs-weather-model/bin:$PATH
   export PYTHONPATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/lib/python3.8/site-packages
-elif [[ $HOSTNAME == *chadmin* ]] || [[ $HOSTNAME == *cheyenne* ]]; then
-  export MACHINE_ID=cheyenne
+elif [[ $HOSTNAME == *chadmin* ]] || [[ $HOSTNAME == *derecho* ]]; then
```

Comment:
Please remove `[[ $HOSTNAME == *chadmin* ]] ||` section as `chadmin` does not exist on Derecho.

Reply:
Done